Source/WarpX.cpp

```diff
@@ -2176,6 +2198,38 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
             dm, ncomps, ngEB, tag("vector_potential_grad_buf_b_stag[z]"), 0.0_rt);
     }
 
+    // Allocate extra multifabs needed by the kinetic-fluid hybrid algorithm.
```

Comment:
If these are members of the `HybridPICModel` class, then the code below will need to be moved to somewhere in the `HybridPICModel` class.